src/ZenithAngleCalc.jl

```diff
@@ -86,7 +83,7 @@ end
 
 function instantaneous_zenith_angle(
     date::DateTime,
-    epoch_string,
+    date0::DateTime,
     (ϖ, γ, e)::Tuple{FT, FT, FT},
```

Comment:
what are these arguments? Are they the same at every lat/long point?
@szy21 do you know?

Reply:
(if they are the same, then we don't need to call `distance_declination_hourangle` at every point.)

Reply:
ϖ, γ, e? They are orbital parameters (longitude of perihelion, obliquity, and eccentricity), and they are the same at every lat/long point. There is some documentation [here](https://clima.github.io/Insolation.jl/dev/ZenithAngleEquations/). I can add some doc string later, but probably not now.